aten/src/ATen/native/native_functions.yaml

```diff
@@ -6945,6 +6945,10 @@
   structured_delegate: triangular_solve.X
   variants: method, function
 
+- func: _linalg_check_errors(Tensor info, str api_name, *, bool is_matrix) -> ()
+  dispatch:
+    CompositeExplicitAutograd: _linalg_check_errors
```

Comment:
Why is this needed?

Reply:
We're hiding the data_ptr calls in a new non-composite operator, `_linalg_check_errors`. Does that answer the question or were you referring to something specific in these three lines?

Reply:
Ho I was was refering to making this CompositeExplicitAutograd vs the default CompositeImplicitAutograd

Reply:
Making this CompositeImplicitAutograd makes it so that this operation is composite with respect to autograd, which is a problem because internally it calls .data_ptr().

Making this CompositeExplicitAutograd makes it so that this operation is not composite with respect to autograd (I think? The naming is really weird) so that the .data_ptr() call is hidden.

Does that sound right?

Reply:
Sounds good.

Reply:
Side note: can we rename CompositeExplicitAutograd? I'm not sure what it's supposed to mean

Reply:
This is the 3rd name for it already....
The idea is:
- CompositeImpliciteAutograd: is a composite op (used by all backends) that is autodiffed (has an implicit autograd).
- CompositeExplicitAutograd: is a composite op (used by all backends) that has an explicit autograd formula.

Reply:
I see, so CompositeExplicitAutograd means it's composite w.r.t. to the backend, not that it is composite w.r.t autograd :P

Reply:
Yes, the composite part is about aten.
implicit/explicit is about autograd.

We did discussed these names at length with Ailing. But it is very very hard to find a name that makes everyone happy :(